### DISCUSSION TITLE
Not able to push my changes to the forked repo

### DISCUSSION THREAD

**Discussion by EvanSingee** [iamparas0/TIC-TAC-TOE#345] [Q&A]
<img width="573" alt="Screenshot 2024-10-21 at 12 10 00 AM" src="https://github.com/user-attachments/assets/ae1abbda-223e-468f-a7ba-43d030e88dae">
I entered PAT when it asked for password

**Comment by EvanSingee** [comment]
@sanjay-kv @iamparas0

**Reply by sanjay-kv** [reply]
Its your auth issue, didnt signed into your account from CLI
